integration/spark/shared/src/main/java/io/openlineage/spark/agent/lifecycle/BaseVisitorFactory.java

```diff
@@ -8,33 +8,7 @@
 import io.openlineage.client.OpenLineage;
 import io.openlineage.client.OpenLineage.Dataset;
 import io.openlineage.client.OpenLineage.InputDataset;
-import io.openlineage.spark.agent.lifecycle.plan.AlterTableAddColumnsCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.AlterTableAddPartitionCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.AlterTableRenameCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.AlterTableSetLocationCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.BigQueryNodeInputVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.BigQueryNodeOutputVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.CreateDataSourceTableAsSelectCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.CreateDataSourceTableCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.CreateHiveTableAsSelectCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.CreateTableCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.DropTableCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.ExternalRDDVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.HiveTableRelationVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.InsertIntoDataSourceDirVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.InsertIntoDataSourceVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.InsertIntoDirVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.InsertIntoHadoopFsRelationVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.InsertIntoHiveDirVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.InsertIntoHiveTableVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.KafkaRelationVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.KustoRelationVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.LoadDataCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.LogicalRDDVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.OptimizedCreateHiveTableAsSelectCommandVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.SqlDWDatabricksVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.SqlExecutionRDDVisitor;
-import io.openlineage.spark.agent.lifecycle.plan.TruncateTableCommandVisitor;
+import io.openlineage.spark.agent.lifecycle.plan.*;
```

Comment:
How come is this passing tests in CI?

Reply:
inteliji is doing me that all the time, I ve changed the threshold for adding an asterisk.